### PR TITLE
Practice app translation

### PR DESCRIPTION
**Pull request by Hryurt**
Please Make a code review and merge if there is not any problem. If there are somethings that must change please write on the linked issue about Translation Endpoint.

Please control these criteria.(If you want you can add extra criteria.)
- [x] Documentation
- [x] Readability
- [x] Code works correctly or not?
- [x] Tests

### CODE REVIEW

File: practice-app/platon_api/platon_api/settings.py [hunk 1]
```diff
@@ -75,9 +78,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    }
+        'ENGINE': 'django.db.backends.mysql', 
+        'NAME': 'localdb',
+        'USER': 'root',
+        'PASSWORD': '=N*~h3,#7G#@yKd',
```
Comment (ertbulbul) [inline practice-app/platon_api/platon_api/settings.py:84]: What kind of password is this

File: practice-app/platon_api/rest_api/translation/test.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        self.assertEqual(response.status_code, 200)
+
+    #Testing for translation of about me with valid input with more than five
+    def test_keyErrorTranslation(self):
+        response = self.client.get("/api/translation/"+self.validToken)
+        self.assertEqual(response.status_code, 200)
+
+    # Testing for translation of about me with unvalid input
+    def test_noTokenInDb(self):
+        response = self.client.get("/api/translation/" + self.unvalidToken)
```
Comment (ertbulbul) [inline practice-app/platon_api/rest_api/translation/test.py:40]: invalid misspelled

**Comment by ahmetdadak** [comment]
Your base seems master branch, try to change it practice-app-development

**Comment by MEHMETTEMIZEL** [comment]
crispy-forms
When I review your code I notice that 'crispy forms' in the settings.py file is not used anywhere. Then I downloaded your code and I try to run your code and test I encounter with the 'ModuleNotFoundError: No module named 'crispy_forms'' . I think you should remove it if it's not necessary. Otherwise great job(eline sağlık).

File: practice-app/platon_api/platon_api/settings.py [hunk 3]
```diff
@@ -31,12 +31,15 @@
...
 INSTALLED_APPS = [
+    'rest_api',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    "rest_framework",
+    "crispy_forms"
```
Comment (MEHMETTEMIZEL) [inline practice-app/platon_api/platon_api/settings.py:42]: I think you should delete this

Comment (MEHMETTEMIZEL) [inline practice-app/platon_api/platon_api/settings.py:42]: I think you should delete this.

Comment (MEHMETTEMIZEL) [reply practice-app/platon_api/platon_api/settings.py:42]: Only "crispy_forms"

File: practice-app/platon_api/platon_api/settings.py (hunk 1 above)
Comment (Hryurt) [reply practice-app/platon_api/platon_api/settings.py:84]: auto generated

File: practice-app/platon_api/rest_api/translation/test.py (hunk 2 above)
Comment (Hryurt) [reply practice-app/platon_api/rest_api/translation/test.py:40]: Thanks, I will immediately resolve it

File: practice-app/platon_api/platon_api/settings.py (hunk 3 above)
Comment (Hryurt) [reply practice-app/platon_api/platon_api/settings.py:42]: We use it anyway, but thanks

**Pull request merged**